storm_ros/GoalMsg goal
---
storm_ros/JointMsg[] joint_data